ui/src/app/intro/views/UserIntro/UserIntro.tsx

```diff
@@ -1,9 +1,105 @@
+import { useEffect } from "react";
+
+import {
+  Button,
+  Card,
+  Icon,
+  List,
+  Row,
+  Spinner,
+} from "@canonical/react-components";
+import classNames from "classnames";
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+
 import Section from "app/base/components/Section";
+import { useWindowTitle } from "app/base/hooks";
+import dashboardURLs from "app/dashboard/urls";
+import machineURLs from "app/machines/urls";
+import authSelectors from "app/store/auth/selectors";
+import { actions as sshkeyActions } from "app/store/sshkey";
+import sshkeySelectors from "app/store/sshkey/selectors";
 
 const UserIntro = (): JSX.Element => {
+  const dispatch = useDispatch();
+  const authLoading = useSelector(authSelectors.loading);
+  const authUser = useSelector(authSelectors.get);
+  const sshkeys = useSelector(sshkeySelectors.all);
+  const sshkeyLoading = useSelector(sshkeySelectors.loading);
+
+  const hasSSHKeys = sshkeys.length > 0;
+
+  useWindowTitle("Welcome");
+
+  useEffect(() => {
+    dispatch(sshkeyActions.fetch());
+  }, [dispatch]);
+
+  if (authLoading || sshkeyLoading) {
+    return <Spinner />;
+  }
+
   return (
-    <Section header="User intro">
-      <div>User intro</div>
+    <Section>
+      <Card
+        data-test="sshkey-card"
+        highlighted
+        title={
+          <span className="p-heading--4">
+            <Icon name={hasSSHKeys ? "success" : "success-grey"} /> SSH keys for{" "}
+            {authUser?.username}
+          </span>
+        }
+      >
+        <p>
+          Add multiple keys from Launchpad and Github or enter them manually.
+        </p>
+        <h4>Keys</h4>
+      </Card>
+      {authUser?.is_superuser ? (
+        <Card
```

Comment:
It seems like the copy in this card only makes sense for the first admin. And why is everything greyed out when you haven't added an SSH key???

whyyyyy

Reply:
Yeah you're totally right, I wonder if this should get removed from here and become part of the flow you're working on. What do you think?

Reply:
That makes more sense imo!

Reply:
OK, I'll remove it from here then.